pyrandwalk/pyrandwalk_obj.py

```diff
@@ -7,6 +7,8 @@
 from numpy import linalg as la
 import networkx as nx
 
+np.random.seed(seed=42)
```

Comment:
what it this for?

Reply:
This was a residue from some tests I ran... my bad 😅